skptesting/benchmark.inc

```diff
@@ -82,6 +82,18 @@ ngx() {
 	pids="$pids $!"
 }
 
+slow() {
```

Comment:
this function is not used, we can remove it

Reply:
done